.github/workflows/ci.yml

```diff
@@ -109,7 +110,7 @@ jobs:
           target: ${{ matrix.target }}
           tags: |
             ${{ steps.login.outputs.registry }}/firezone/${{ matrix.image_name }}:${{ env.VERSION }}-${{ github.sha }}
-            ${{ steps.login.outputs.registry }}/firezone/${{ matrix.image_name }}:latest
+            ${{ steps.login.outputs.registry }}/firezone/${{ matrix.image_name }}:${{ env.BRANCH_TAG }}
```

Comment:
The integration-tests just below this use the `docker-compose.yml` which reference the image tags here I think, so we may still need a `latest`?

Reply:
Or I supposed that's buggy too. The real solution I suppose would be to `sed` the `docker-compose` images with our version tag (for now), or even better -- use a different compose or just raw `docker run` commands for the integration tests.

Reply:
I think a proper solution would be to read the VERSION tag and set it on CI rather than relying on the `latest` label. For the docker-compose file - we should use `main` branch tag by default.

Reply:
Pushed a fix, see https://github.com/firezone/firezone/pull/2315/commits/9ccd963e7db497c132a63e619af3aa47ad3038c2